config/routes.rb

```diff
@@ -1,13 +1,18 @@
 Rails.application.routes.draw do
   devise_for :users
+  root 'users#index'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
   get '/public_recipes', to: 'recipes#public_recipes'
+
+  resources :foods, only: [:index, :new, :create]
```

Comment:
You did a great job creating routes to food controller actions 👍, but following the ERD below, foods belong to a user, so I guess `resources :foods, only: [:index, :new, :create]` should be nested in `resources :users, only: %i[index show]` as we did for `resources :recipes, only: %i[index show update]`. What do you think?

![image](https://github.com/momo-87/recipe-app/assets/122116375/491496dc-3a40-47fd-bc38-c4facf9aedea)